### ISSUE TITLE
Add sound_type enum class values

### ISSUE THREAD

**Issue by richelbilderbeek** [tresinformal/game#72]
There is an enum class called `sound_type` in `sound_type.h`:

```c++
enum class sound_type
{
  // Sound types here
};
```

Add the `sound_type`s so that the following test passes:

```c++
void test_sound_type()
{
  #ifdef FIX_ISSUE_72
  assert(sound_type::shoot != sound_type::hit);
  #endif
}
```

You are allowed to add some extra sound types!

**Comment by richelbilderbeek** [comment]
I will unassign this Issue. Volunteers are welcome :+1:

**Comment by Neves-P** [comment]
I suspect this was fixed by @sebmader in ed861f0
Is there anything else needed?

**Comment by richelbilderbeek** [comment]
It's done :+1:
